suites/multimds/basic/mount/fuse/default-permissions/yes.yaml

```diff
@@ -0,0 +1,5 @@
+overrides:
+  ceph:
+    conf:
+      client:
+        fuse default permissions: true
```

Comment:
Uh, how does adding this new folder of selections work with also testing against the kclient? I'm pretty sure it'll out the kclient frag in every test along with one of these fuse ones, and that's no good?